playwright/playwright.config.ts

```diff
@@ -40,15 +35,15 @@ export const cluster = process.env.SM_ENV === "staging" ? "exp" : undefined;
 export const auth = {
   username: process.env.PLAYWRIGHT_ADMIN_USERNAME,
   password: process.env.PLAYWRIGHT_ADMIN_PASSWORD,
-  storageState: ".auth/admin.json",
 };
 
 assert.ok(auth.username, "Missing PLAYWRIGHT_ADMIN_USERNAME env variable.");
 assert.ok(auth.password, "Missing PLAYWRIGHT_ADMIN_PASSWORD env variable.");
 
+export const REPOSITORY_NAME_PREFIX = "e2e-tests-";
```

Comment:
what is this for? can't be confusing whether you have to provide a repository with a name like this or without the prefix becuase it's added later?

Reply:
It's not really for "you" meaning dev usage, it's purely for the CI and if you want the system to create a repo for you.
If you provide your own repo name, that's it you already have it, no need to manage that.

Do you think the naming should be different then? You have an idea?

Reply:
i think it's unnecessary, i.e. we don't care how the repo is named or if it's named with that particular prefix, unless i'm missing something

the fact that you have to export it from here to use it somewhere else starts smelling a bit

but in the end it's not super important either, i understand now its meaning and that's enough for me, i think the name is ok too - but again, i don't see why it's important that the repositories have this particular prefix, doesn't add anything unless i'm missing something

Reply:
It's an engineering decision to have all repositories created dynamically for e2e tests to have this pattern.
It helps us to recognize them easily for SRE, debug, maintenance, etc.